lib/puppet/transaction/report.rb

```diff
@@ -351,9 +351,9 @@ def summary
 
       report[key].keys.sort { |a,b|
         # sort by label
-        if a == TOTAL
+        if a == :total
```

Comment:
what happened here? with this change, we no longer have `total` appearing at the end of the report

```
Time:
         Schedule: 0.00
          Package: 0.00
   Property list key: 0.00
          Printer: 0.01
             Cron: 0.01
             File: 0.07
          Service: 0.10
   Profile manager: 0.31
   Convert catalog: 0.46
             Exec: 1.66
   Transaction evaluation: 11.75
   Catalog application: 11.86
         Last run: 1522792065
      Plugin sync: 3.64
            Total: 30.50
   Config retrieval: 4.43
   Node retrieval: 4.54
   Fact generation: 5.55
        Pe anchor: 0.00
       Filebucket: 0.00
``

is there a reason you reverted this change?